### PR TITLE
resolves #14126, add preference option to show search panel at  bottom

### LINKED ISSUE #14126: Move search panel at bottom
We need a extension that is capable to move Search Pannel at bottom .
Same like the extension that move git pannel at bottom .
Somebody can make this ?

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

**Comment by navch** [comment]
@ad1992 Can you sign the CLA located at http://dev.brackets.io/brackets-contributor-license-agreement.html?

**Comment by ad1992** [comment]
done @navch . The tests are passing now.

**Comment by navch** [comment]
@ad1992 I think there is one problem. Please try opening Quick Open (Ctrl + Shift + O). Or click on drop down icon in Search box. This might not be the good experience.

image

**Comment by ad1992** [comment]
@navch I have fixed the above issue. Can you please check now.

**Comment by navch** [comment]
@ad1992 Cool Stuff. Working like a charm 👍

File: package.json [hunk 1]
```diff
@@ -73,4 +73,4 @@
             "url": "https://github.com/adobe/brackets/blob/master/LICENSE"
         }
     ]
-}
\ No newline at end of file
+}
```
Comment (navch) [inline package.json]: May be we can revert this file.

**Comment by swmitra** [comment]
@ad1992 Great work 👍 

In Brackets we show all document commands UI in a top anchored modal. That's how most of our users are familiar in seeing such functionalities. Although, with a preference this looks acceptable, but then for users, who wants to see such functionalities in a bottom panel should see every document command UI as bottom panels. So in order to make this a reality, I guess the whole UI paradigm has to be configurable via preference instead of a chosen few. 

If you are ready to make such widespread changes with supporting tests, it can be part of the core. But if it's targeted to one or two of such functionalities, I guess it would server better as an extension. Brackets DOM is extendible enough to be customized from an extension by just using css/less. 

Please let us know if you need any such pointers.

**Comment by ad1992** [comment]
Thank you @navch @swmitra  for looking into this. I had actually discussed with @petent about the approach before creating the PR. I agree that for better user experience all document command UI should be shown as bottom panels. Since this would take some time so I can build an extension for solving the issue referenced in the PR.  If this feature helps then I would love to work on it as a core and making changes across all document command UI as well. But may be should we wait for such request from the users and then start working on core ? what do you suggest ?

**Pull request closed without merging**